pkg/generate/app/cmd/newapp.go

```diff
@@ -561,7 +552,17 @@ func (c *AppConfig) RunQuery() (*QueryResult, error) {
 
 func (c *AppConfig) validate() (cmdutil.Environment, cmdutil.Environment, error) {
 	var errs []error
-
+	// Environment File where the parameters are stored
+	if len(c.EnvironmentFile) != 0 {
+		varList, err := app.GetEnvironmentVars(c.EnvironmentFile)
+		if err != nil {
+			return nil, nil, fmt.Errorf("Error reading file %v: %v", c.EnvironmentFile, err)
+		} else {
+			for _, env := range varList {
+				c.Environment = append(c.Environment, env)
```

Comment:
Idea looks good - as of now, it is throwing a warning saying that value in --env is being overwritten(Meaning the precedence is other way around-first file and then cmd line arg). I am ok with changing it other way around.

The simplest I can think of to implement this behavior will be to in c.Environment if the item already exists. If it exists don't append or else append.